openapi.yaml

```diff
@@ -33,53 +106,58 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Thread'
+              $ref: '#/components/schemas/CreateThreadRequest'
       responses:
         '201':
```

Comment:
Right now this is actually a 200, but I can update on the backend to a 201 since that is technically more correct